chsdi/static/vectorStyles/ch.bafu.hydroweb-messstationen_gefahren.json

```
@@ -208,20 +279,53 @@
     },
     {
       "geomType": "point",
-      "value": 2,
+      "value": 2.2,
+      "minResolution": 250,
+      "vectorOptions": {
+        "type": "circle",
+        "radius": 6,
+        "fill": {
+          "color": "rgba(255,255,  0,1)"
+        },
+        "stroke": {
+          "color": "rgba(117,117,117,1)",
+          "width": 2
+        }
+      }
+    },
+    {
+      "geomType": "point",
+      "value": 2.2,
+      "minResolution": 100,
+      "maxResolution": 250,
+      "vectorOptions": {
+        "type": "circle",
+        "radius": 7,
+        "fill": {
+          "color": "rgba(255,255,  0,1)"
+        },
+        "stroke": {
+          "color": "rgba(117,117,117,1)",
+          "width": 2
+        }
+      }
+    },
+    {
+      "geomType": "point",
+      "value": 2.2,
       "maxResolution": 100,
       "vectorOptions": {
         "type": "circle",
         "radius": 8,
         "fill": {
-          "color": "rgba(255,255,0,1)"
+          "color": "rgba(255,255,  0,1)"
```

```suggestion
          "color": "rgba(255,255,0,1)"
```
not sure if it is relevant but if it is there are a couple more like this below.